2-mandatory/3-financial-times.js

```diff
@@ -5,16 +5,25 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
-    // TODO
+       if(allArticleTitles.lengh !== 0){
+        allArticleTitles = ARTICLE_TITLES.filter((el) => el.length <= 65);
+        return allArticleTitles;
+    
+}
+else return allArticleTitles=[];
 }
 
 /*
     The editor of the FT likes short headlines with only a few words!
     Implement the function below, which returns the title with the fewest words.
     (you can assume words will always be seperated by a space)
 */
+
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+  let arr = allArticleTitles;
+  let newArr = arr.sort((a, b) => a.length - b.length);
```

Comment:
good use of sort here, to sort an array without mutating the original array you can use the spread operator: `let newArr = [...allArticleTitles].sort((a, b) => a.length - b.length);`. We could  use a different array name here to make the code more readable, for example changing `newArr` to something like `fewestWordsArr`